docs/operations/runbooks/incident-response.md

```
@@ -0,0 +1,105 @@
+
+**Quick fixes:**
+- Toggle feature flag to disable problematic code:
+  ```javascript
+  // In Statsig console, disable gate
+  // Or set env: FEATURE_X_ENABLED=false
+  ```
+- Scale resources if capacity issue
+- Rollback deployment if recent change caused issue
+
+**Rollback procedure:**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)